src/pages/Facility/services/pharmacy/CreateDispenseSheet.tsx

```
@@ -240,6 +264,22 @@ export function CreateDispenseSheet({
 
+  // When patientId is provided, render a simple button that navigates directly
+  if (patientId && preselectedPatient) {
+    const triggerElement = trigger || (
+      <Button>
+        <Plus className="size-4 mr-1" />
+        {t("new_dispense")}
+      </Button>
+    );
+
+    return (
+      <span onClick={handleDirectDispense} className="cursor-pointer">
```

Biome (2.3.13)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)